README.md

```diff
@@ -1,5 +1,5 @@
 <div align="center">
-	<a href="https://autocomplete.algolia.com"><img src="./media/illustration.png" alt="Autocomplete" width="200" height="200"></a>
+	<a href="https://www.algolia.com/doc/ui-libraries/autocomplete/introduction/what-is-autocomplete"><img src="./media/illustration.png" alt="Autocomplete" width="200" height="200"></a>
 	<h1>Autocomplete</h1>
 	<p>
 		<strong>A JavaScript library that lets you quickly build autocomplete experiences</strong>
```

Comment:
I noticed the npm shield is outdated (stuck on alpha.29), not sure why.

Reply:
`alpha.29` was the first alpha version that we released with that name, but npm treats the first published version as the `latest` tag. So this number will bump once we release stable.